src/locale/fr-ca.js

```diff
@@ -32,6 +32,7 @@ export default moment.defineLocale('fr-ca', {
         future : 'dans %s',
         past : 'il y a %s',
         s : 'quelques secondes',
```

Comment:
To be consistent, the `s` should be `une minute` (and apply to the other strings as well)

Reply:
The `duration-second` in Unicode CLDR 32.0 ([link](http://www.unicode.org/repos/cldr/tags/release-32/common/main/fr_CA.xml)) is `{0} à la seconde`.

@yanickrochon Would both `s` and `ss` be `une minute` in `fr-ca`?

Reply:
No, because this is an acceleration unit, and the "duration-second"'s `{0}` parameter is a prefix, not a numeric value. Much as "100 miles per gallon" is, where the localization string "duration-second"  would be "{0} per gallon", and the prefix is "100 miles".

E.g. "{0} à la second" (i.e. "{0} per second") translates to "0 mètre à la seconde" or "2 mètres à la seconde" (notice the plural for the last one vs singlar for the first.

In French, the 0 is always singular, unlike in English.(see [plurals and ordinals for languages](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html))

If you want to cover the proper internationalization for an interval of [0..1.999*] seconds, the proper string "s" would be "une seconde", and "ss" would be "%d secondes".

(**Note:** in French, we usually use "à l'instant", meaning "just now", when the event happened less than a few seconds ago. But this would be invariant. Just like "now", you cannot say "now ago", neither "il y a à l'instant". Thus, "s" would technically need to be invariant for certain langauges.)

Reply:
I just realized that I wrote a typo in my first comment, I wrote "minute" instead of "second". Sorry for the confusion.